_pages/dev/ssr/server-side-rendering-error-handling.md

```diff
@@ -0,0 +1,349 @@
+---
+title: SSR Error Handling
+feature:
+  - name: SSR Error Handling
+    spa_version: 2211.29
+    cx_version: n/a
+---
+
+SSR Error Handling functionality provides the missing behavior to how Angular engine handles errors during rendering pages on the server. It ensures the Spartacus application reacts on the encountered errors by providing default set of tools to handle them and, at the same time, giving a possibility to customize the experience. This feature is crucial for the SEO of the page and the overall UX.
+
+**Note:** Angular SSR by default doesn't ignore an error only if occurred in the APP_INITIALIZER or synchronous bootstrap of the root component.
+
+New Spartacus apps created since v2211.29, have the SSR Error Handling enabled by default.
```

Comment:
```suggestion
New Spartacus apps created since v2211.29, have the SSR Error Handling enabled by default, which addresses the limitations of OOTB Angular SSR.
```